build.gradle

```diff
@@ -1,13 +1,19 @@
+apply plugin: "de.fayard.buildSrcVersions"
+
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.3.72"
     repositories {
         google()
         jcenter()
+        gradlePluginPortal()
+        maven { url 'https://jitpack.io' }
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-alpha01"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath Libs.com_android_tools_build_gradle
+        classpath Libs.kotlin_gradle_plugin
+        classpath Libs.de_fayard_buildsrcversions_gradle_plugin
+        classpath Libs.detekt_gradle_plugin
+        classpath Libs.hilt_android_gradle_plugin
 
         // NOTE: Do not place your application dependencies here; they belong
```

Comment:
I personally get rid of the extra noise from these comments, not sure if you like to do the same :)

Reply:
Can do yeah, as I don't visit this file often I generally don't care 😂